lib/puppet/module.rb

```diff
@@ -68,6 +68,17 @@ def initialize(name, path, environment)
     load_metadata
 
     @absolute_path_to_manifests = Puppet::FileSystem::PathPattern.absolute(manifests)
+
+    # i18n initialization for modules
+    if Puppet::GETTEXT_AVAILABLE
```

Comment:
`GETTEXT_AVAILABLE` as a top-level constant for a feature test feels a little weird.  The ship has probably sailed on this decision, but wondering why this isn't in a feature?